opm/common/OpmLog/LogBackend.cpp

```diff
@@ -27,18 +27,33 @@ namespace Opm {
     {
     }
 
+    LogBackend::~LogBackend()
+    {
+    }
 
-    bool LogBackend::includeMessage(int64_t messageFlag) {
-        if (((messageFlag & m_mask) == messageFlag) &&
-            (messageFlag > 0))
-            return true;
-        else
-            return false;
+    void LogBackend::configureDecoration(std::shared_ptr<MessageFormatterInterface> formatter)
+    {
+        formatter_ = formatter;
```

Comment:
Don't get me wrong: there's not much wrong about `this->` (apart from being a bit wordy in contexts where we invoke multiple members), and I don't think your arguments are without merit. You missed my point however: nothing enforces using `this->`, if I write `data = 7` that is perfectly valid. Do you have external links providing further arguments? (That's what I asked for...)

In any case, I'll enforce the existing rule where I am maintainer, elsewhere other rules will be followed (such as `m_` in opm-parser I guess), we'll just have to accept this and live with it. Keeping things consistent within modules is good, and within single files mandatory. As long as everybody's able to adapt as needed I think there is no big problem.